webcomponents/tool/src/main/frontend/package.json

```diff
@@ -55,7 +55,7 @@
     "eslint-plugin-html": "^8.1.2",
     "eslint-plugin-lit": "^1.15.0",
     "eslint-plugin-wc": "^2.2.0",
-    "fetch-mock": "^12.0.2",
+    "fetch-mock": "^9.11.0",
```

Comment:
purposeful?

Reply:
Yes it was on purpose. The tests are broken because the latest fetch mock needs to be imported in a different fashion. Serious breaking changes between 9 and 12.